app/assets/stylesheets/layouts/_navbar.css.scss

```diff
@@ -43,18 +43,33 @@
 .navbar-brand {
   font-weight: 600;
   @include transition(all .2s ease-out);
+  @extend .hvr-underline-reveal;
+  fill: #C2B8AF;
+  stroke: #C2B8AF;
+}
 
-  &:hover{
-    text-shadow: 0 0 10px lighten($right-1-color, 20%),
-                 0 0 20px lighten($right-1-color, 20%),
-                 0 0 30px lighten($right-1-color, 20%),
-                 0 0 40px lighten($right-1-color, 20%),
-                 0 0 70px lighten($right-1-color, 20%),
-                 0 0 80px lighten($right-1-color, 20%),
-                 0 0 100px lighten($right-1-color, 20%),
-                 0 0 150px lighten($right-1-color, 20%);
-    // color: lighten($right-1-color, 60%);
-  }
+.navbar-brand:hover{
+  fill: #FAFDFE;
+  stroke: #FAFDFE;
+}
+
+#logo {
```

Comment:
Would something like `.logo-container` for the `div` and `.logo` for the `img` make more sense?